src/instruments/view/instrumentpaneltreemodel.cpp

```diff
@@ -344,6 +346,7 @@ int InstrumentPanelTreeModel::columnCount(const QModelIndex&) const
 
 QVariant InstrumentPanelTreeModel::data(const QModelIndex& index, int role) const
 {
+    TRACEFUNC;
```

Comment:
I think that better to remove TRACEFUNC from data and isSelected because those methods are trivial

Reply:
By the way, I wanted to see how many times this method is called. Two instruments, one (piano) to expanded, two more elements appear - 59 calls to this method :)

Reply:
removed